### DISCUSSION TITLE
n1 armbian 官方文档装不上

### DISCUSSION THREAD

**Discussion by jiejianshiwaxinge** [v2rayA/v2rayA#1528] [Q&A]
我是n1 Armbian 使用v2raya官网debian/ubuntu安装文档，第一二步骤添加公钥那里就报错了，死活装不上，又尝试手动安装v2raya.dea包,但是geoip文件提示缺少，这个文件自动下载不成功(n1已经处于科学网络状态)，添加公钥报错如何解决呢？请问有有经验的n1 Armbian v2raya大佬吗?  使用这个一键脚本可以成功安装吗？
sudo sh -c "$(wget -qO- https://github.com/v2rayA/v2rayA-installer/raw/main/installer.sh)" @ --with-xray

**Comment by gsssz** [comment]
换源  update 更新系统 这几个操作做了么？是root执行的么？ 我在比特米盒上没这个问题

**Reply by f1161291** [reply]
公钥那里安装 不上是需要手动创建一个文件夹
